### PR TITLE
Add an Sv locale

### PR DESCRIPTION
**Pull request by olleolleolle**
The Sv locale is the same as the SvSE one, the translations of the words added here do not differ in other Swedish-speaking regions.

### CODE REVIEW

**Comment by AlexWayfer** [comment]
I'm a bit confused that there are no spec changes, also that there is YAML files, not only `.rb`, and you haven't changed them.

Based on YAML files for English, we can (should?) rename `base/sv-se.yml` to `sv.yml`

I can try to double-check.

**Comment by AlexWayfer** [comment]
Also fixed some bugs (with date formatting).

And removed `en` as a sublocale of Swedish.

Added specs, specified titles, etc.

I hope everything is right. If something — feel free to write about it.

File: lib/r18n-core/locales/sv.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        sublocales: %w[sv-SE],
+
+        wday_names: %w[söndag måndag tisdag onsdag torsdag fredag lördag],
+        wday_abbrs: %w[sön mån tis ons tor fre lör],
+
+        month_names: %w[
+          januari februari mars april maj juni juli augusti september oktober
+          november december
+        ],
+        month_abbrs: %w[jan feb mar apr maj jun jul aug okt nov dec],
```
Comment (olleolleolle) [inline lib/r18n-core/locales/sv.rb:18]: Sep is missing here, after aug.

Comment (AlexWayfer) [reply lib/r18n-core/locales/sv.rb:18]: Thank you. I'll fix this (with specs) before a new release.

**Pull request merged**